projects/plugins/protect/src/js/components/pricing-table/index.jsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import {
+	Button,
+	ProductPrice,
+	PricingTable,
+	PricingTableColumn,
+	PricingTableHeader,
+	PricingTableItem,
+} from '@automattic/jetpack-components';
+import { useConnection } from '@automattic/jetpack-connection';
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import useAnalyticsTracks from '../../hooks/use-analytics-tracks';
+import useProtectData from '../../hooks/use-protect-data';
+
+/**
+ * Product Detail component.
+ *
+ * @param {object} props                     - Component props
+ * @param {Function} props.onSecurityAdd     - Callback when adding paid protect product successfully
+ * @param {Function} props.securityJustAdded - Callback when adding paid protect product was recently added
+ * @returns {object}                ConnectedPricingTable react component.
+ */
+const ConnectedPricingTable = ( { onSecurityAdd, securityJustAdded } ) => {
+	const { siteIsRegistering, handleRegisterSite, registrationError } = useConnection( {
+		skipUserConnection: true,
+	} );
+
+	// Access paid protect product data
+	const { securityBundle } = useProtectData();
+	const { pricingForUi } = securityBundle;
+
+	// Compute the price per month.
+	const price = pricingForUi.fullPrice
+		? Math.ceil( ( pricingForUi.fullPrice / 12 ) * 100 ) / 100
+		: null;
+	const offPrice = pricingForUi.discountPrice
+		? Math.ceil( ( pricingForUi.discountPrice / 12 ) * 100 ) / 100
+		: null;
+	const { currencyCode: currency = 'USD' } = pricingForUi;
+
+	// Track free and paid click events
+	const { recordEventHandler } = useAnalyticsTracks();
+	const getSecurityBundle = recordEventHandler(
+		'jetpack_protect_pricing_table_get_security_link_click',
+		onSecurityAdd
+	);
+	const getProtectFree = recordEventHandler(
+		'jetpack_protect_connected_product_activated',
+		handleRegisterSite
+	);
+
+	const args = {
```

Comment:
These tooltips still have the placeholder content, which is funny, because I imagine if you remove the placeholders the info icons won't show up.

In the Figma design, the icons are on the left side and appear to just be for show instead of displaying any additional info via tooltip. I'm wondering if we would we be able to remove the tooltip info and pass in the icon inside the `name` property? Like so...
```
name: (
    <>
        <Icon icon={ whateverTheInfoIconNameIs } />
        { __( 'Stay one step ahead of threats', 'jetpack-protect' ) }
    </>
),
```
What do you think?

Reply:
Right, the current Figma designs for Protect don't show that the tooltip icons are functional but based on other P2 discussions, it was my impression that if the icon is there, it should display some additional detail about the feature. I think we concluded previously, that we may not yet have that content so placeholders might be fine for the time being. However, in an effort to ensure we don't accidentally ship unfinished code, I have no problem pursuing this alternative. @miguelxpn would you agree or do you think there is someone we could wrap in to supply the detail we need for each feature tooltip?

Reply:
I think it's fine to remove them for now and then we make a follow up to get exactly what content we want to show there and address it.